app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transferdetails/components/TransferInfo.kt

```diff
@@ -49,8 +49,8 @@ import com.infomaniak.swisstransfer.ui.utils.PreviewLightAndDark
 @Composable
 fun TransferInfo(transfer: TransferUi) {
 
-    val filesCount = transfer.files.count()
-    val downloadedCount by remember { derivedStateOf { transfer.downloadLimit - transfer.downloadLeft } }
+    val filesCount  by remember(transfer) { derivedStateOf { transfer.files.count() } }
```

Comment:
You should not use a derivedStateOf if there's no significant optimization. 

In other words, if you have an event "A" and you have a value you want to compute "B" which value depends on A. If A changing to a different value implies that B will change to a different value as well, then you don't want to use a derivedStateOf. You want to use a derivedStateOf when A changing to a different value can lead to B staying the same value.

Reply:
This conversation is marked as resolved, but the issue seems to still be here.

Is it on purpose ?

Reply:
After a discussion with @sirambd and @LunarX , we're keeping this as is.